Signal/src/AppDelegate.m

```diff
@@ -178,11 +178,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(databaseViewRegistrationComplete)
-                                                 name:kNSNotificationName_DatabaseViewRegistrationComplete
+                                                 name:DatabaseViewRegistrationCompleteNotification
                                                object:nil];
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(registrationStateDidChange)
-                                                 name:kNSNotificationName_RegistrationStateDidChange
+                                                 name:RegistrationStateDidChangeNotification
```

Comment:
Renaming notifications to conform to their naming convention so that they can be accessed from Swift.